airflow/contrib/operators/file_to_wasb.py

```diff
@@ -57,8 +57,8 @@ def execute(self, context):
         """Upload a file to Azure Blob Storage."""
         hook = WasbHook(wasb_conn_id=self.wasb_conn_id)
         self.log.info(
-            'Uploading %s to wasb://%s '
-            'as %s'.format(self.file_path, self.container_name, self.blob_name)
+            'Uploading {} to wasb://{} '
+            'as {}'.format(self.file_path, self.container_name, self.blob_name)
```

Comment:
Was this changes because of a flake8 error ? We use `%s` format for logs so that the arguments are passed properly when logging levels are changed

Reply:
Yes. This was because of flake8 errors.

Reply:
@kaxil any thoughts on what to do here? change them or set ignore flags?

Reply:
also if you're running .format on the string before it's even parsed does it even matter?

Reply:
Coz of https://github.com/apache/airflow/pull/4804#issuecomment-468594870

Reply:
@kaxil fixed

Reply:
The fix here is to do

```python
        self.log.info(
            'Uploading %s to wasb://%s as %s',
            self.file_path, self.container_name, self.blob_name)
```

That way if the logging level is disabled, no time is wasted formatting the string

Reply:
@ashb If possible I'd like to avoid making changes unrelated to the commit on this PR (especially since it's going straight to 1-10). Should we make a separate PR for fixing these logs?

Reply:
Oh gotcha. I mean you are already changing this, so it's going to conflict anyway.

Potato potato.